src/Data/Nat/Properties.agda

```diff
@@ -243,6 +243,9 @@ n≤1+n _ = ≤-step ≤-refl
 n≤0⇒n≡0 : ∀ {n} → n ≤ 0 → n ≡ 0
 n≤0⇒n≡0 z≤n = refl
 
+1+m≤1+n⇒m≤n : ∀ {m n} → 1 + m ≤ 1 + n → m ≤ n
```

Comment:
FYI this is a special case of both:
* `pred-mono`
* `+-cancelˡ-≤`

Reply:
Or indeed it's simply `≤-pred`